BookPlayer/Utils/Extensions/VerticalLabelStyle.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  VerticalLabelStyle.swift
+//  BookPlayer
+//
+//  Created by Gianni Carlo on 22/8/25.
+//  Copyright © 2025 BookPlayer LLC. All rights reserved.
+//
+
+import SwiftUI
+
+struct VerticalLabelStyle: LabelStyle {
```

Comment:
I think it could be great to have preview at least for those reusable component.